pkg/ccl/backupccl/generative_split_and_scatter_processor.go

```diff
@@ -291,6 +308,7 @@ func runGenerativeSplitAndScatter(
 	importSpanChunksCh := make(chan scatteredChunk, splitWorkers*2)
```

Comment:
I should've done this in your previous PR sorry, but do you think you could add a line or two above each goroutine in this method describing what each of them does?